gips/data/merra/merra.py

```diff
@@ -103,7 +102,7 @@ class merraAsset(Asset):
             'shortname': 'M2T1NXSLV',
             'description': '2d,1-Hourly,Time-Averaged,Single-Level,Assimilation,Single-Level Diagnostics V5.12.4',
             'url': 'https://goldsmr4.gesdisc.eosdis.nasa.gov/data/MERRA2/M2T1NXSLV.5.12.4',
-            'pattern': _asset_pattern.format(name='tavg1_2d_slv_Nx'),
+            'pattern': r'MERRA2_\d\d\d\.tavg1_2d_slv_Nx\.\d{4}\d{2}\d{2}\.nc4',
```

Comment:
Seems like this should continue the `_asset_re_pattern.format(name='tavg1_2d_slv_Nx')` style, for DRYness.